src/tripCard/TripCard.unit.tsx

```diff
@@ -1,161 +1,225 @@
+    expect(screen.getByText('Alone in the back')).toBeInTheDocument()
+  })
+
+  it('Should not have the max two icon by default', () => {
+    const props = createProps({
+      titles: { maxTwo: 'Max two' },
+    })
+
+    render(<TripCard {...props} />)
+    expect(screen.queryByText('Max two')).not.toBeInTheDocument()
   })
 
-  it('Should render metaUrl if provided', () => {
-    const tripCard = shallow(<TripCard {...mockedProps} metaUrl={null} />)
-    expect(tripCard.find('meta')).toHaveLength(0)
+  it('Should have the max two icon', () => {
+    const props = createProps({
+      flags: { maxTwo: true },
+      titles: { maxTwo: 'Max two' },
+    })
 
-    const tripCardWithMetaUrl = shallow(<TripCard {...mockedProps} metaUrl="blablacar.fr" />)
-    expect(tripCardWithMetaUrl.find('meta')).toHaveLength(4)
+    render(<TripCard {...props} />)
+    expect(screen.getByText('Max two')).toBeInTheDocument()
+  })
+
+  it('Should not have the auto approval icon by default', () => {
+    const props = createProps({
+      titles: { autoApproval: 'Auto approval' },
+    })
+
+    render(<TripCard {...props} />)
+    expect(screen.queryByText('Auto approval')).not.toBeInTheDocument()
+  })
+
+  it('Should have the auto approval icon', () => {
+    const props = createProps({
+      flags: { autoApproval: true },
+      titles: { autoApproval: 'Auto approval' },
+    })
+
+    render(<TripCard {...props} />)
+    expect(screen.getByText('Auto approval')).toBeInTheDocument()
+  })
+
+  it('Should not render meta tags by default', () => {
+    const props = createProps()
+    render(<TripCard {...props} />)
+    expect(screen.queryByRole('meta')).not.toBeInTheDocument()
+  })
+
+  it('Should render meta tags if provided', () => {
+    const props = createProps({ metaUrl: 'blablacar.fr' })
+    const { container } = render(<TripCard {...props} />)
+
+    expect(
+      container.querySelector('meta[itemprop="url"][content="blablacar.fr"]'),
+    ).toBeInTheDocument()
+    expect(
+      container.querySelector('meta[itemprop="name"][content="Paris → Bordeaux"]'),
+    ).toBeInTheDocument()
+    expect(
+      container.querySelector('meta[itemprop="startDate"][content="2017-12-11T09:00"]'),
+    ).toBeInTheDocument()
+    expect(
+      container.querySelector('meta[itemprop="endDate"][content="2017-12-11T12:00"]'),
+    ).toBeInTheDocument()
   })
 
   it('Should show driver avatar', () => {
-    const driver = {
-      avatarUrl: '//placehold.it/500x500',
-      firstName: 'Jane',
-    }
+    const props = createProps({
+      driver: {
+        avatarUrl: '//placehold.it/500x500',
+        firstName: 'Jane',
+      },
+    })
+
+    render(<TripCard {...props} />)
+    expect(screen.getByAltText('Jane')).toBeInTheDocument()
+  })
 
-    const component = shallow(<TripCard {...mockedProps} driver={driver} passengers={null} />)
-    expect(component.find(Avatar)).toHaveLength(1)
+  it('Should render driver sub text', () => {
```

Comment:
Nothing wrong but it made me realize we were not testing for the ratings

Reply:
I added one